schemas/WebpackOptions.json

```diff
@@ -2132,6 +2132,60 @@
         },
         {
           "$ref": "#/definitions/PerformanceOptions"
+        },
+        {
+          "type": "array",
+          "items": {
+            "description": "Configuration entry for web performance recommendations.",
+            "type": "object",
+            "additionalProperties": false,
+            "properties": {
+              "assetFilter": {
+                "description": "Filter function to select assets that are checked",
+                "instanceof": "Function",
+                "tsType": "Function"
+              },
+              "hints": {
+                "description": "Sets the format of the hints: warnings, errors or nothing at all",
+                "enum": [false, "warning", "error"]
+              },
+              "maxAssetSize": {
+                "description": "Filesize limit (in bytes) when exceeded, that webpack will provide performance hints",
+                "type": "number"
+              },
+              "maxEntrypointSize": {
+                "description": "Total size of an entry point (in bytes)",
+                "type": "number"
+              }
+            }
+          }
+        },
+        {
+          "anyOf": [
+            {
+              "type": "object",
```

Comment:
This is repeated from https://github.com/webpack/webpack/blob/16a7adf04bcd2bbcb19ed394a6402297d2b1b74b/schemas/WebpackOptions.json#L1073-L1095